README.md

```diff
@@ -0,0 +1,63 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid Metrics
+
+This is a mobile web application that features 2 pages: The homepage and the details page. The homepage displays all today's statistics about confirmed cases, deaths, and recoveries caused by Covid-19 in the world, as a total and separated by countries. When the user clicks on any country, the website navigates to the details page, which contains the detailed statistics about that specific country as a total and separated by regions. Both pages contain a search bar that allows the user to filter by country name. The data is consumed from [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). Built with React, Redux, and Bootstrap.
+
+## Mobile
+
+![screenshot](./app_screenshot.png)
+
+## Desktop
+
+![screenshot](./app_screenshot-2.png)
+
+## Built With
+
+## Live Demo
+
+[Live Demo Link](https://dicodiaz-covid-metrics.netlify.app)

```

Comment:
Currently, the deployed app is not working properly, Kindly fix this 👍

Reply:
- [ ] [REQUIRED] As recommended by the previous reviewer, the Netlify deployment is still not yet successful. Kindly check the log files to see where the deployment has failed.